cpp/unreal_plugins/UrdfBot/Source/UrdfBot/UrdfRobotComponent.cpp

```diff
@@ -0,0 +1,152 @@
+//
+// Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+//
+
+#include "UrdfRobotComponent.h"
+
+#include "CoreUtils/Config.h"
+#include "CoreUtils/Unreal.h"
+#include "UrdfJointComponent.h"
+#include "UrdfParser.h"
+
+UUrdfRobotComponent::UUrdfRobotComponent(const FObjectInitializer& object_initializer): USceneComponent(object_initializer)
+{
+    FString urdf_file_path = FPaths::Combine(Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_DIR")), Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_FILE")));
```

Comment:
Fit within our new line width limit of 158 chars for easier browsing on GitHub.

```
    std::string urdf_file = Unreal::toString(FPaths::Combine(
        Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_DIR")),
        Unreal::toFString(Config::get<std::string>("URDFBOT.URDFBOT_PAWN.URDF_FILE"))));
    UrdfRobotDesc robot_desc = UrdfParser::parse(urdf_file_path);
```

Reply:
fixed